resources/development-guidelines/good-commit-practice.md

```diff
@@ -360,7 +360,7 @@ The main rule to follow is:
 
 > The commit message must contain all the information required to fully
 > understand & review the patch for correctness.
-> **Less is _not_   more.More is more.**
+> **Less is *not*   more.More is more.**
```

Comment:
```suggestion
> **Less is *not* more. More is more.**
```
Just a suggestion for updating this sentence where spacing looks more appealing.